katas/test/src/pokerhands/juan/PlayedHandTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿using CardJuan;
+using HandJuan;
+using PlayedHandJuan;
+
+namespace PlayedHandTestsJuan
+{
+    public class PlayedHandTests
+    {
+        [Fact]
+        public void TestGetStraigthFlush()
+        {
+            string opt = "";
+            string expected = "AH2H3H4H5H";
+            foreach (Card card in PlayedHand.GetStraightFlush(new Hand("2H 3H 4H 5H AH")))
+            {
+                opt += card.ValueAttr.ToString() + card.SuitAttr.ToString();
+            }
+
+            Assert.Equal(expected, opt);
+        }
+
+        [Fact]
+        public void TestGetFourOfAKind()
+        {
+            string opt = "";
```

Comment:
what `opt` stands for?

Reply:
Fixed this issue by properly renaming the variable name

Reply:
Thanks